start-local.sh

```diff
@@ -369,9 +374,35 @@ wait_for_kibana() {
 
 EOM
   fi
+
   cat >> start.sh <<- EOM
+if [ ! -n "\${ES_LOCAL_LICENSE:-}" ] && [ "\$today" -gt $expire ]; then
+  echo "---------------------------------------------------------------------"
+  echo "The trial period has expired, you can continue with the Basic license"
+  echo "or ask to extend the trial for another 30-day using this form:"
+  echo "https://www.elastic.co/trialextension"
```

Comment:
This is just a suggestion, but take it or leave it. I was thinking of new users who wouldn't necessarily read the README, where the license information is pretty clear under [Features](https://github.com/elastic/start-local#-features). If I just copy the code from elastic.co and run it, I don't necessarily know that I'm using a trial license (or I forgot after using it for a month). Initially I thought it would make sense to repeat that information here, but maybe just mentioning the script comes with a trial license again would be enough. 

```suggestion
  echo "This script comes with a one-month trial license."
  echo
  echo "The trial period has expired, you can continue using it with the Free"
  echo " and open - Basic license or ask to extend the trial for another"
  echo " 30-day using this form:"
  echo "https://www.elastic.co/trialextension"
```

Reply:
Thanks @picandocodigo for the review, I propose to change the sentence as follows:
`The one-month trial period has expired. You can continue using the Free and open Basic license or request to extend the trial for another 30 days using this form:`
WDYT?

Reply:
Sounds good to me :+1: